internal/nitrite_test.go

```diff
@@ -32,164 +31,124 @@ func TestNitrite_Verify(t *testing.T) {
 	)
 	require.NoError(t, err)
 
-	attestatations := map[string]struct {
+	tests := []struct {
+		name         string
 		attestation  []byte
 		time         time.Time
 		certProvider internal.CertProvider
 		allowDebug   bool
 	}{
-		"nitro": {
+		{
+			name:        "happy path - nitro",
 			attestation: nitroAttestation,
 			time:        internal.NitroAttestationTime,
 			certProvider: internal.NewNitroCertProvider(
 				internal.NewEmbeddedRootCertZipReader(),
 			),
 		},
-		"debug": {
+		{
+			name:        "happy path - debug nitro",
 			attestation: debugNitroAttestation,
 			time:        internal.DebugNitroAttestationTime,
 			certProvider: internal.NewNitroCertProvider(
 				internal.NewEmbeddedRootCertZipReader(),
 			),
 			allowDebug: true,
 		},
-		"self-signed": {
+		{
+			name:         "happy path - self signed",
 			attestation:  selfSignedAttestation,
 			time:         internal.SelfSignedAttestationTime,
 			certProvider: internal.NewSelfSignedCertProvider(),
 		},
 	}
 
-	for key := range attestatations {
-		t.Run("happy path", func(t *testing.T) {
-			t.Log(key)
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
 
 			// when
 			result, err := internal.Verify(
-				attestatations[key].attestation,
-				attestatations[key].certProvider,
+				tt.attestation,
+				tt.certProvider,
 				internal.WithAttestationTime(),
-				attestatations[key].allowDebug,
+				tt.allowDebug,
 			)
 
 			// then
 			require.NoError(t, err)
 			// make sure at least one of the fields is populated and attested
 			assert.Equal(
 				t,
-				attestatations[key].time.UTC(),
+				tt.time.UTC(),
 				result.Document.CreatedAt().UTC(),
 			)
 		})
 
-		t.Run("cannot get root certificates", func(t *testing.T) {
```

Comment:
Moved all certificate related tests to `internal/document_test.TestDocument_CheckCertificates()`